service-deployment/bootstrap/grafana/grafana.yaml.template

```diff
@@ -0,0 +1,52 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: grafana
+  namespace: monitor
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        task: monitor
+        k8s-app: grafana
+    spec:
+      hostNetwork: true
+      hostPID: true
+      nodeSelector:
+        grafana: "true"
+      containers:
+      - name: grafana
+        image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}grafana
+        imagePullPolicy: Always
+        ports:
+        - containerPort: {{ clusterinfo['grafanainfo']['grafana_port'] }}
+          protocol: TCP        
+        env:
+        - name : GRAFANA_URL
+          value: {{ clusterinfo['grafanainfo']['grafana_url'] }}
+          # The following env variables are required to make Grafana accessible via
+          # the kubernetes api-server proxy. On production clusters, we recommend
+          # removing these env variables, setup auth for grafana, and expose the grafana
```

Comment:
The grafana in PAI is accessed by IP or kubernetes service?

Reply:
we can access by IP and don't need service, I have removed the grafana service .